app/models/orphan.rb

```diff
@@ -66,6 +66,8 @@ class Orphan < ActiveRecord::Base
   accepts_nested_attributes_for :current_address, allow_destroy: true
   accepts_nested_attributes_for :original_address, allow_destroy: true
 
+  default_scope { includes(:partner, :orphan_sponsorship_status, :orphan_status, original_address: :province) }
```

Comment:
Would this work for filters: http://activeadmin.info/docs/2-resource-customization.html#customizing_resource_retrieval?